packages/blocks/src/latex-block/latex-block.ts

```diff
@@ -114,11 +114,7 @@ export class LatexBlockComponent extends CaptionedBlockComponent<
     this._editorAbortController?.abort();
     this._editorAbortController = new AbortController();
 
-    this.selection.setGroup('note', [
-      this.selection.create('block', {
-        blockId: this.model.id,
-      }),
-    ]);
+    this.selection.setGroup('note', []);
```

Comment:
I think you should stop propagation of the latex editor popup or add another hotkey handler which returns `true` to avoid potential keymap conflict instead of do something like this.